flexget/plugins/urlrewrite_newpct.py

```diff
@@ -20,27 +29,56 @@ class UrlRewriteNewPCT(object):
 
     # urlrewriter API
     def url_rewritable(self, task, entry):
-        url = entry['url']
-        rewritable_regex='^http:\/\/(www.)?newpct1?.com\/.*'
-        return re.match(rewritable_regex,url) and not url.startswith('http://www.newpct.com/descargar/')
+        if entry.get('urls'):
```

Comment:
urls[0] should always be the same as the regular url, so I don't think this is needed. Especially since the actual rewrite function doesn't handle it.